cppcache/src/ThinClientPoolDM.cpp

```diff
@@ -392,7 +388,7 @@ void ThinClientPoolDM::manageConnections(std::atomic<bool>& isRunning) {
     if (isRunning) {
       manageConnectionsInternal(isRunning);
       while (m_connSema.tryacquire() != -1) {
-        ;
+        // ignored
```

Comment:
Since this just creates a hot loop, why not just call blocking version of `acquire`?

Reply:
Same issue found in a few other files.

Reply:
Some day I'll learn to read my changes properly instead of blindly fixing warnings!